pyanaconda/modules/storage/platform.py

```diff
@@ -224,7 +224,7 @@ def _bootloader_partition(self):
         return PartSpec(
             mountpoint="/boot/efi",
             fstype="efi",
-            size=Size("200MiB"),
+            size=Size("500MiB"),
             max_size=Size("600MiB"),
```

Comment:
With the default bumped all the way to 500,  keeping 600 for the maximum is not really facilitating much enlargement. IOW should max_size also be increased, perhaps to 1GiB ?

Reply:
Yes, agreed, I've modified the patch to allow scaling up to 2GiB now.